static/docs/shinobiplus.md

```diff
@@ -4,7 +4,7 @@
 
 Shinobi Chess is a chess variant designed in 2021 by Couch Tomato with help from Fables, fourth in the series of asymmetric games. The Western chess army (“the Kingdom,” black) has invaded the land of the Sakura Clan (pink/sakura). While having mostly weak pieces at the start, the Clan is very resourceful and can instantly recruit and summon allies to defend at a minute’s notice. The Clan starts with many pieces in hand and can drop them on its side of the board in lieu of moving a piece. Furthermore, they can promote upon reaching the far end of the board – these abilities are similar to the game of shogi, with the major exception that captured pieces do not go into a player’s hand, so each drop counts! 
 
-The game was initially considered balance by engine selfplay. However, player experience and also deeper NNUE engine skill revealed that the pink army had quite a decent advantage. Therefore, "Shinobi+" was created as a more balanced version. Most pieces were moved to being in the hand, and the new Fox piece was introduced. Additionally, black was allowed an earlier promotion rank.
+The game was initially considered balanced by engine selfplay. However, player experience and also deeper NNUE engine skill revealed that the pink army had quite a decent advantage. Therefore, "Shinobi+" was created as a more balanced version. Most pieces were moved to being in the hand, and the new Fox piece was introduced. Additionally, black was allowed an earlier promotion rank.
```

Comment:
I think "also" should also be cut. "player experience and deeper NNUE engine skill revealed ..."